servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/CoreMessageSender.java

```diff
@@ -536,16 +459,14 @@ public void onError(Exception completionException)
     }
 
     @Override
-    public void onSendComplete(final Delivery delivery)
-    {
+    public void onSendComplete(final Delivery delivery) {
         DeliveryState outcome = delivery.getRemoteState();
-        final String deliveryTag = new String(delivery.getTag());
+        final String deliveryTag = new String(delivery.getTag(), UTF_8);
```

Comment:
[SpotBugs P1] Reliance on default encoding: Found a call to a method which will perform a byte to String (or String to byte) conversion, and will assume that the default platform encoding is suitable.